src/main/resources/assets/javascripts/components/RunsTable.jsx

```diff
@@ -208,6 +212,22 @@ function selectQuery(query, e) {
   QueryActions.selectQuery(query);
 }
 
+function selectTable(table, e) {
+  e.preventDefault();
+  TableActions.addTable({
+    name: table
+  });
+  TableActions.selectTable(table);
+  TabActions.selectTab(TabConstants.DATA_PREVIEW);
+}
```

Comment:
Oh I see, this is a pattern you follow. Disregard my comment above. I'm not against it, just haven't seen this before. I guess since these functions don't use `this` from the component this is fine.